.github/workflows/release.yaml

```diff
@@ -262,11 +262,20 @@ jobs:
           echo "Deleting branch $branch_name"
         done
 
-    - name: Publish Release Event
+    - name: Publish Release Event to Website
       if: inputs.release_candidate == false
       uses: peter-evans/repository-dispatch@v3
       with:
         token: ${{ steps.generate_token.outputs.token }}
         repository: open-component-model/ocm-website
         event-type: ocm-cli-release
         client-payload: '{"tag": "${{ env.RELEASE_VERSION }}"}'
+
+    - name: Publish Release Event for AUR
```

Comment:
That step triggers the Release for AUR and chocolatey or am I mistaken? If it triggers both I would adjust the `name`

Reply:
done